### PR TITLE
Change to a prettier font

### PR DESCRIPTION
**Pull request by sercero**

### CODE REVIEW

**Comment by paroj** [comment]
"prettier" is of course quite subjective. However when changing the Font the following should be considered:
- use imgui_overlay.addFont, so Ogre can increase the font resolution according to the pixel_ratio
- use a monospace font for the log, as some outputs rely on that to be aligned

File: ogre_mesh_viewer.py [hunk 1]
```diff
@@ -555,8 +555,8 @@ def setup(self):
         # for picking
         self.ray_query = scn_mgr.createRayQuery(Ogre.Ray())
 
-        imgui_overlay.addFont("SdkTrays/Value", RGN_MESHVIEWER)
-        self.logwin.font = ImGui.GetIO().Fonts.AddFontDefault()
+        imgui_overlay.addFont("Liberation/Sans", Ogre.RGN_DEFAULT)
+        self.logwin.font = imgui_overlay.addFont("Liberation/Mono", Ogre.RGN_DEFAULT)
         self.logwin.font.Scale = round(pixel_ratio)
```
Comment (paroj) [inline ogre_mesh_viewer.py:560]: this is no longer needed for ttf fonts

File: Liberation.fontdef [hunk 2]
```diff
@@ -0,0 +1,15 @@
+font Liberation/Sans
```
Comment (paroj) [inline Liberation.fontdef:1]: please move the new files to a fonts/ directory

File: ogre_mesh_viewer.py (hunk 1 above)
Comment (sercero) [reply ogre_mesh_viewer.py:560]: I don't understand. What is no longer needed?

File: Liberation.fontdef (hunk 2 above)
Comment (sercero) [reply Liberation.fontdef:1]: Perhaps they could go in a zip file

File: ogre_mesh_viewer.py (hunk 1 above)
Comment (paroj) [reply ogre_mesh_viewer.py:560]: Scale = round(pixel_ratio)

File: Liberation.fontdef (hunk 2 above)
Comment (paroj) [reply Liberation.fontdef:1]: actually, I would prefer a folder for easier editing & git history

**Comment by paroj** [comment]
can you rebase on master, so the test is run?

**Comment by sercero** [comment]
> can you rebase on master, so the test is run?

I did...
The test did run

**Comment by paroj** [comment]
woops.. did not notice that. However it actually crashes:
https://github.com/OGRECave/ogre-meshviewer/actions/runs/8038772015/job/21955059980#step:4:270

seems I did not properly propagate the return value..

**Comment by sercero** [comment]
> woops.. did not notice that. However it actually crashes: https://github.com/OGRECave/ogre-meshviewer/actions/runs/8038772015/job/21955059980#step:4:270
> 
> seems I did not properly propagate the return value..

What would be the best way to add the application or the font directory to a resource group, perhaps using "./fonts" is not the proper way...

**Comment by paroj** [comment]
> seems I did not properly propagate the return value..

rebase again..

> What would be the best way to add the application or the font directory

os.path.dirname(__file__+"/fonts")

**Comment by sercero** [comment]
> > seems I did not properly propagate the return value..
> 
> rebase again..

Strange... the CI is not running it seems

**Pull request merged**